### ISSUE TITLE
A11y - Label should have associated control

### ISSUE THREAD

**Issue by Rahat-ch** [Rahat-ch/The_Sylar_Project#36]
Currently for labels in the form we are getting the following listing error:
A form label must be associated with a control.eslint(jsx-a11y/label-has-associated-control)

Need to refactor the code to have the label associated with a control. This may require some changes on styling in addition to structural changes. See the documentation below for an idea on where to start:

https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/master/docs/rules/label-has-associated-control.md

Remove the eslint disable in components/NewResourceOverlay/form.js to see the error

**Comment by JDHofmann** [comment]
Happy to grab this one!
